include/zcbor_common.h

```diff
@@ -420,7 +420,10 @@ bool zcbor_compare_strings(const struct zcbor_string *str1,
  *
  *  @return  The length of the header in bytes (1-9).
  */
-size_t zcbor_header_len(size_t num_elems);
+size_t zcbor_header_len_64(uint64_t value);
+
+/** Like @ref zcbor_header_len_64 but for integer of any size <= 8. */
+size_t zcbor_header_len(const void *const value, size_t value_len);
```

Comment:
I guess the major version should be bumped at this point? Or are we still in the 0.x playground? I would strongly suggest keeping the existing function as is and instead having a sub optimal name for the newly added API.

Reply:
Thinking about it, I agree. Changed it back, and named the new function `zcbor_header_len_ptr`.